src/base-element.js

```diff
@@ -134,6 +135,14 @@ export class BaseElement {
     this.resources_ = resourcesFor(this.getWin());
   }
 
+  /**
+  * Returns the element priority. A value from 0 to 10.
```

Comment:
@dvoytenko Would you mind commenting here with a comment that would be appropriate for this method regarding the semantics of the returned number. Thanks!